src/components/shared/FilterHelper.js

```diff
@@ -0,0 +1,154 @@
+import React from "react"
+import {
+    FormLabel,
+    FormGroup,
+    TextField
+} from '@material-ui/core'
+import { RESTRICTED_COL_INDEX } from './ColumnHelper'
+
+export const createRangeFilter = (labelName, isPriceRangeFilter) => {
+    isPriceRangeFilter = isPriceRangeFilter || false
+    if(isPriceRangeFilter || false) return createPriceRangeFilter(labelName)
```

Comment:
Personally I think it would make more sense to just have the filter options be separate exports, rather than have an argument to decide which one is returned here. Then consumers just import whichever they need instead of importing one thing that might eventually require a bunch of arguments they don't care about. Not super relevant since we're dropping the price from here, just a general comment.